pubspec.yaml

```diff
@@ -9,17 +9,21 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  intl:
+  intl_translation:
   cupertino_icons: ^0.1.3
   community_material_icon: ^3.5.95
 
-  flutter_toolbox: ^4.4.3
+  flutter_toolbox: ^4.4.4
 
-  http: ^0.12.0+3
+  http: ^0.12.0+4
 
   localstorage: ^3.0.1+4
   shared_preferences: ^0.5.6
   flutter_secure_storage: ^3.3.1+1
   local_auth: ^0.6.1
+  timeago: ^2.0.24
```

Comment:
By the way, we can use it also to show time in transactions history? (separate PR)